build.gradle

```diff
@@ -150,7 +150,6 @@ project(':datastream-server-api') {
   dependencies {
     compile project(':datastream-common')
     compile project(':datastream-utils')
-    compile "com.linkedin.kafka.clients:li-apache-kafka-clients:$LIKafkaVersion"
```

Comment:
Just curious, why is this line getting removed instead of getting replaced with "com.linkedin.kafka:kafka-clients..."

Reply:
There is no Open source version of this package. Its completely moved inside of Li. That is why we need to remove the dependency on this package. Will add more details to this rb. Thanks